src/pages/en/guides/server-side-rendering.md

```diff
@@ -119,12 +119,13 @@ if(!product) {
 
 #### API Routes
 
-A [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) can also be returned from an API route.
+An API route is a .js or .ts file in the src/pages/ folder that takes a request and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Comment:
```suggestion
An API route is either a `.js` or `.ts` file within the `src/pages/` folder which  takes a [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a `[Response]`(https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Reply:
Just a small formatting nit

Reply:
```suggestion
An API route is a `.js` or `.ts` file within the `src/pages/` folder that takes a [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a `[Response]`(https://developer.mozilla.org/en-US/docs/Web/API/Response).
```

Aaand.... this is what I would do! (I don't think "either" needs to be there, and I feel like "that" is more in the voice of these docs than "which.")

Reply:
English is a romantic language

“Speak the speech, I pray you, as I pronounc'd it to you, trippingly on the tongue.”